docs/pages/server-access/guides/tsh.mdx

```diff
@@ -482,3 +483,19 @@ This is possible even if nodes in the `production` cluster are located behind a
 firewall without open ports. This works because the `production` cluster
 establishes a reverse SSH tunnel back into `work` proxy, and this tunnel is used
 to establish inbound SSH connections.
+
+## Configuration file
```

Comment:
You can probably get away with removing this section and just keeping the version on the CLI page.

Reply:
I agree. Or at least make it a partial/include and reuse in both places so we don't have to copy/paste.

Reply:
Went with just having the CLI page.